### PR TITLE
[bitnami/kube-prometheus] Support prometheus-operator namespace allowlist. (#13525)

### LINKED ISSUE #13525: Support kube-prometheus co-existence with the prometheus-operator in 'openshift-monitoring'
### Name and Version

bitnami/kube-prometheus

### What is the problem this feature will solve?

If you deploy the kube-prometheus chart to an Openshift cluster already running RH's cluster-monitoring functionality - you end up with 2 prometheus-operators fighting of the same kind:Prometheus and you break the existing openshift-controlled prometheus-etc. instances.

### What is the feature you are proposing to solve the problem?

Expose the existing operator's `-namespace` argument in the helm chart. Not specifying a value causes the existing behaviour (no argument is added, all namespaces are watched.), specifying a value injects the `-namespaces` argument to the operator, and causes it to watch the specific NS only (allowlist)

### What alternatives have you considered?

operator.args _could_ be used here, but then I have to keep all the the other currently-templated-args under my control - it is a hammer to crack a nut.
Better, have the chart support the upstream operator configurable explicitly.

### PR DESCRIPTION
**Pull request by mlashley**
### Description of the change

Support specifying the namespaces that the prometheus-operator watches.

### Benefits

Chart can co-exists with 'Openshift Monitoring' (or presumably multiple operator-versions in a single cluster)

### Possible drawbacks

None known 

### Applicable issues

  - fixes #13525

### Additional information

First time PR here - hope I got this right. I will be AFK for a couple of weeks from the end of the week - so I haven't died if we don't get this in before then.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/kube-prometheus/Chart.yaml [hunk 1]
```diff
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/prometheus
   - https://github.com/bitnami/containers/tree/main/bitnami/alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 8.1.12
+version: 8.1.13
```
Comment (marcosbc) [inline bitnami/kube-prometheus/Chart.yaml:38]: Since this is adding a new feature, could you increase the minor version instead? (e.g. to `8.2.0`)

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please have a look at my comments whenever you have a chance.

Once you have checked them (and fixed any issue if applicable), please reply back with an individual comment, for the response to be properly tracked by our support system.

File: bitnami/kube-prometheus/Chart.yaml (hunk 1 above)
Comment (mlashley) [reply bitnami/kube-prometheus/Chart.yaml]: Sure thing - I figured this was 50/50 bugfix/feature.

**Comment by mlashley** [comment]
Thanks for the review @marcosbc  - all comments attended to.

**Review by marcosbc — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**